apps/web/src/components/forms/monitor/section-requests.tsx

```diff
@@ -40,11 +41,21 @@ interface Props {
 // TODO: add Dialog with response informations when pingEndpoint!
 
 export function SectionRequests({ form }: Props) {
-  const { fields, append, remove } = useFieldArray({
+  const { fields, append, prepend, remove, update } = useFieldArray({
     name: "headers",
     control: form.control,
   });
+
   const watchMethod = form.watch("method");
+  const [content, setContent] = useState<string>("application/json");
```

Comment:
Instead of adding a new state, we could use

```
const watchHeaders = form.watch('headers')
const contentType = watchHeaders.find(header => ... === 'Content-Type')
```